pkg/portfwdserver/server.go

```diff
@@ -1,69 +1,84 @@
 package portfwdserver
 
 import (
+	"context"
 	"errors"
 	"io"
 	"net"
+	"time"
 
+	"github.com/containers/gvisor-tap-vsock/pkg/tcpproxy"
 	"github.com/lima-vm/lima/pkg/guestagent/api"
 )
 
-type TunnelServer struct {
-	Conns map[string]net.Conn
-}
+type TunnelServer struct{}
 
 func NewTunnelServer() *TunnelServer {
-	return &TunnelServer{
-		Conns: make(map[string]net.Conn),
-	}
+	return &TunnelServer{}
 }
 
 func (s *TunnelServer) Start(stream api.GuestService_TunnelServer) error {
```

Comment:
TunnelServer is an empty type now. Why does it need to exist? This can be a free function.

Reply:
I agree but just for usecases like before, it is useful to retain that empty. So that later if we want to support reverse forwarding we can use these with different methods / some caching etc.